src/bootstrap/build.rs

```diff
@@ -3,6 +3,8 @@ use std::path::PathBuf;
 
 fn main() {
     println!("cargo:rerun-if-changed=build.rs");
+    println!("cargo:rerun-if-env-changed=RUSTC");
+    println!("cargo:rerun-if-env-changed=PATH");
```

Comment:
PATH seems correct, but I am surprised that RUSTC is needed: cargo [docs](https://doc.rust-lang.org/cargo/reference/build-scripts.html#rerun-if-env-changed) claim that those variables set by Cargo are already taken care of by Cargo: "...it is not necessary to use this for environment variables like TARGET that Cargo sets."

Maybe RUSTC is missing there? It would be great if you could file an issue upstream, since presumably that's the cause?

Reply:
> It would be great if you could file an issue upstream, since presumably that's the cause?

ok, but I suspect it will be a while before it's fixed

Reply:
yeah, happy to r+ this before then, just wanted something tracking things. I don't think the fix is that important.